blocks/header-nav-chain-block/chains/header-nav-chain-block/default.jsx

```diff
@@ -138,7 +138,8 @@ const Nav = (props) => {
 
   const navHeight = desktopNavivationStartHeight || 56;
 
-  const showDotSeparators = showHorizontalSeperatorDots || true;
+  const showDotSeparators = (showHorizontalSeperatorDots === null
+      || showHorizontalSeperatorDots === undefined) ? true : showHorizontalSeperatorDots;
```

Comment:
```suggestion
  const showDotSeparators = showHorizontalSeperatorDots ?? true;
```

My attempt to be more concise. Am I reading this right?